runners/google-cloud-dataflow-java/pom.xml

```diff
@@ -288,11 +288,21 @@
     </dependency>
 
     <dependency>
+      <groupId>org.apache.avro</groupId>
```

Comment:
Previously there was no direct dependency on Avro/Datastore in Dataflow runner. This change adds integration tests which directly reference classes in these packages.